### DISCUSSION TITLE
Welcome to Random-Disco-Light-Simulator Discussions!

### DISCUSSION THREAD

**Discussion by sk66641** [sk66641/Random-Disco-Light-Simulator#52] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded 💪.

  To get started, comment below with an introduction of yourself .

**Comment by piyushsahujtp** [comment]
I this you need to make  a disscussion for the website blog to make people more know about your project , explaining them how this project works.
